tools/debugger/test/check-order.js

```diff
@@ -16,13 +18,24 @@ describe('Orders', async () => {
   // Owns a crypto kitty
   const kittyWallet = '0x7F18BB4Dd92CF2404C54CBa1A9BE4A1153bdb078'
 
+  // Owns a 'creature' NFT
+  const nftWallet = '0xA916A5830d21bc05C4c56Ce5452Cc96D8edD8f8c'
+
+  // Owns a 20 tokens of ID 5 in the ERC1155 token contract
+  const erc1155Wallet = '0x65c2Fe7C4Ef4d6E8f2eB2aC8A116af04566bF490'
+
   // Mock ERC721Receiver wallet on rinkeby
-  const erc721Wallet = '0xF727956c4CFd20b9C8D463218a65e751891da3e6'
+  const erc721Receiver = '0xF727956c4CFd20b9C8D463218a65e751891da3e6'
+
+  // Mock ERC1155Receiver wallet on rinkeby
+  const erc1155Receiver = '0x55791bA427a47807FBC0E4D61e7908Db217FC475'
 
   // rinkeby addresses
   const ASTAddress = '0xcc1cbd4f67cceb7c001bd4adf98451237a193ff8'
   const WETHAddress = '0xc778417e063141139fce010982780140aa0cd5ab'
   const cryptoKittiesAddress = '0x16baf0de678e52367adc69fd067e5edd1d33e3bf'
+  const creatureNFTAddress = '0x12adf24b46b05ea3cb13b8a7d96579966055946f'
+  const mintable1155Address = '0x90fF319Eed5606110A12A0BCfD2C6258352a740e'
 
   const rinkebySwap = '0x43f18D371f388ABE40b9dDaac44D1C9c9185a078'
```

Comment:
This should be updated to the most recent rinkeby Swap contract? The one listed does not support ERC1155. 0x2e7373D70732E0F37F4166D8FD9dBC89DD5BC476?